src/DataManagerPlugin.cs

```diff
@@ -0,0 +1,88 @@
+using static System.Linq.Enumerable;
+using Bep = BepInEx;
+using CG = System.Collections.Generic;
+using HL = HarmonyLib;
+using IO = System.IO;
+using Json = Newtonsoft.Json;
+
+namespace Silksong.DataManager;
+
+[Bep.BepInAutoPlugin(id: "org.silksong-modding.datamanager")]
+public partial class DataManagerPlugin : Bep.BaseUnityPlugin
+{
+    // This property will never be accessed before Start executes.
+    internal static DataManagerPlugin Instance { get; private set; } = null!;
+
+    internal CG.Dictionary<string, IOnceSaveDataMod> onceSaveDataMods = new();
+
+    private void Awake()
+    {
+        Instance = this;
+        Log.Logger = Logger;
+        new HL.Harmony(Id).PatchAll();
+
+        Log.Info("Mod Loaded");
```

Comment:
This is redundant, because we already log each mod that uses save data in Start.